WooCommerce/Classes/Model/Note+Woo.swift

```diff
@@ -66,3 +66,34 @@ private extension Note {
         static let emptyStar    = "\u{2606}"  // Unicode White Star ☆
     }
 }
+
+extension Note {
+    var orderID: Int? {
+        return meta.identifier(forKey: .order)
+    }
+
+    func markOrderNoteAsReadIfNeeded() {
```

Comment:
Added this helper method to centralize marking note as read.
Basically we check if the order id of this Note and the synced one is the same, if it is we mark it as read.